### ISSUE TITLE
Mysql integrity violation issue

### ISSUE THREAD

**Issue by prapraj** [Flagbit/Magento-FilterUrls#22]
I have used this extension in one of the live site which was ok at first  but later gave sql integrity violation error with duplicate entry in 'guia_pe-1',  UN_FILTERURL_REWRITE_STORE_STORE. It would be great if anyone has fix for this

**Comment by DuckThom** [comment]
I know this is a very old issue but since we just came across this error on one of our old M1 sites, here is what fixed it for us.

The length of the `rewrite` column in the `filterurls_rewrite` table is 40, however we have some attribute values that are longer than 40 characters causing them to overlap with existing entries.

For example, if 'book' and 'boot' were to be trimmed to only 3 characters, then you'd have the same issue.

TL;DR, this query fixed it:

```sql
ALTER TABLE `filterurls_rewrite` MODIFY `rewrite` VARCHAR(100) NOT NULL DEFAULT ''
```
